data/sql/updates/pending_db_world/fixmechimmunemaskspectperf.sql

```diff
@@ -0,0 +1,4 @@
+--
+UPDATE `creature_template`
+SET `mechanic_immune_mask` = 650854267, `flags_extra` = 256
+WHERE `entry` = 16473 AND `name` = 'Spectral Performer';
```

Comment:
No need to use that many lines, but it isn't exactly bad, mind you, just not usual
Other than that when dealing with bitmask you need to use bitmask operations, | for add, &~ for subtraction
And name isn't needed because creature_template entries are immutable
```suggestion
--
UPDATE `creature_template` SET `mechanic_immune_mask` = `mechanic_immune_mask`|2|16|64|256|512|1024|2048|4096|8192|131072|524288|4194304|8388608|33554432, `flags_extra` = `flags_extra`|256 WHERE `entry` = 16473;
```